tests/test_utils.cpp

```diff
@@ -345,6 +366,12 @@ void compareModel(const libcellml::ModelPtr &m1, const libcellml::ModelPtr &m2)
         auto c2 = m2->component(index);
         compareComponent(c1, c2, m2);
     }
+
+    for (size_t index = 0; index < m1->importSourceCount(); ++index) {
+        auto i1 = m1->importSource(index);
+        auto i2 = m2->importSource(index);
+        compareImportSource(i1, i2, m2);
```

Comment:
what happens if `m1` and `m2` have different numbers of import sources?

Reply:
m2 is the cloned model from m1.

Reply:
I guess we could assert they're the same first ...

Reply:
2385edb6